images/mod-tile/start.py

```diff
@@ -169,26 +154,26 @@ def expire_tiles(state_file_path, currently_expired_tiles_file_path, rendered_st
             raise
         else:
             if start <= end:
-                expired_tiles = unique_tiles_from_files(start, end, EXPIRED_DIR)
+                expired_tiles = unique_tiles_from_files(start, end, EXPIRED_DIRECTORY)
                 update_currently_expired_tiles_file(currently_expired_tiles_file_path, expired_tiles)
                 if len(expired_tiles) > 0:
-                    render_expired(currently_expired_tiles_file_path)
+                    render_expired(currently_expired_tiles_file_path, action_id)
+                    log.info('marked expired tiles', extra={'action_id': action_id})
                 update_rendered_state_file(rendered_state_file_path, end)
         finally:
             action_id += 1
             time.sleep(RENDER_EXPIRED_TILES_INTERVAL)
 
 
-def render_expired(currently_expired_tiles_file_path):
+def render_expired(currently_expired_tiles_file_path, action_id):
     """
     Render expired tiles
     Args:
         currently_expired_tiles_file_path (str): path to a file with a list of expired tiles to be rendered
     """
-    # TODO: consider extracting map, min-zoom and touch-from env var
-    run_subprocess_async(
-        fr'cat {currently_expired_tiles_file_path} | /src/mod_tile/render_expired --map=osm --min-zoom=8 --touch-from=8 >/dev/null', log.info, log.error, True)
-    log.info('rendered expired tiles')
+    command = fr'cat {currently_expired_tiles_file_path} | /src/mod_tile/render_expired --map=osm --min-zoom={TILE_EXPIRE_MIN_ZOOM} --touch-from={TILE_EXPIRE_MIN_ZOOM} >/dev/null'
+    thread = run_command_async(command, process_log.info, process_log.error, completed_unsuccessfully, completed_successfully)
+    thread.join()
```

Comment:
Why not use `run_command`? you use the `async` version of `run_command` but then immediately invokes the `join` method on the `thread`

Reply:
done